app.py

```diff
@@ -54,6 +54,23 @@
 MAX_RETRIES = 3
 
 
+async def hits_with_rules(host_info: dict):
+    """Populate hits list with rule_id and details."""
+    hits = []
+    rules = host_info['recommendations'].get(
+        'applicable_rules', [AI_SERVICE.replace("-", "_")]
+    )
+    for rule in rules:
+        rule_id = f'{rule}|{rule.upper()}'
+        hits.append(
```

Comment:
If you'd desire shorter code, you can do this as
```suggestion
        hits.append(dict(rule_id=rule_id, details=host_info))
```
```suggestion
        hits.append({'rule_id': rule_id, 'details': host_info})
```
If it fits on one line it should be favored. Personally I also think it's a bit more readable. :wink:

Reply:
if it can fit in 1 line without pylama complaining... I'm all for it! 😉

Reply:
done